src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
```

Comment:
The class 'Figure' should not be abstract if it's only used to set behavior for classes. Consider whether you need an abstract class or if an interface would be more appropriate for defining behaviors like 'AreaCalculator' and 'Drawable'.